stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewGroupDescriptor.java

```diff
@@ -82,13 +100,60 @@ public void unhook() {
       }
     }
 
-    public Object getChildAt(ViewGroup element, int index) {
-      if (index < 0 || index >= element.getChildCount()) {
+    public void setIsDecorView(boolean value) {
```

Comment:
nit: `markDecorView` which always sets it to true.  It makes it clear that you're just locating it, and that you won't be unsetting this value once marked.  It's less flexible, but on purpose.